src/build.py

```diff
@@ -1258,6 +1264,9 @@ def Summary(repos):
   print 'Failed steps: %s.' % buildbot.Failed()
   for step in buildbot.FailedList():
     print '    %s' % step
+  print 'Warned steps: %s.' % buildbot.Warned()
+  for step in buildbot.WarnedList():
+    print '    %s' % step
 
   if IsBuildbot():
     latest_file = '%s/%s' % (BUILDBOT_BUILDERNAME, 'latest.json')
```

Comment:
That sounds kinda iffy, because then we'd have Green builds not updating LKGR. We could introduce Yellow builds, but then why not just have Red builds?

Alternatively: we make JSC failures errors, but let tests run when there are errors? Although that depends on the broader dependency-handling mechanism.

Alternatively: we hack/hardcode this to not upload LKGR if JSC failed (while turning the bot red here?), with a TODO to make that more general.